vaccine_feed_ingest/runners/wa/state/parse.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+
+import json
+import pathlib
+import sys
+
+from vaccine_feed_ingest.utils.log import getLogger
+
+logger = getLogger(__file__)
+
+
+if sys.argv[1] is None:
+    logger.error("Must pass an output_dir as second argument")
+    sys.exit(1)
+
+if sys.argv[2] is None:
+    logger.error("Must pass an input_dir as second argument")
+    sys.exit(1)
+
+input_dir = pathlib.Path(sys.argv[2])
+output_dir = pathlib.Path(sys.argv[1])
+output_file = output_dir / "data.parsed.ndjson"
+
+results = []
+for input_file in input_dir.glob("*.json"):
+    with input_file.open() as fin:
+        data = json.load(fin)
+        try:
+            locations = data["data"]["searchLocations"]["locations"]
+        except KeyError:
```

Comment:
minor: Instead of checking for a `KeyError` can you check each field and then log the error for that field. That will make this easier to fix later if this breaks.

I know this is more verbose, but it definitely will hep with debugging later.

Example:
```
if not data:
    log.error("No data!")
    continue
   
if not data.get("data"):
    log.error("No data data!")
    continue

if not data["data"].get("searchLocations"):
    log.error("No data data searchLocations!")
    continue
    
if not data["data"]["searchLocations"]["locations"]:
    log.warning("No data data searchLocations locations!")
    continue
```

Reply:
That's great, thanks. Done